Rover/sailboat.cpp

```diff
@@ -239,18 +246,36 @@ void Sailboat::get_throttle_and_mainsail_out(float desired_speed, float &throttl
     if (!is_positive(desired_speed)) {
         mainsail_out = 100.0f;
     } else {
-        // Sails are sheeted the same on each side use abs wind direction
+        switch (sail_control_type) {
+
+            // standard linear interpolation control
+            case (SailControlType::LINEAR): {
```

Comment:
```suggestion
            case (SailControlType::LINEAR):
            defualt:  {
```
Just so we have constant behavior if users set a random value